client/hosting-overview/components/plan-card.tsx

```diff
@@ -224,16 +224,33 @@ const PlanCard: FC = () => {
 						</>
 					) }
 				</div>
-				{ ! isStaging && ! isAgencyPurchase && (
+				{ ! isStaging && (
 					<>
-						<PricingSection />
+						{ isAgencyPurchase && (
+							<div className="hosting-overview__plan-agency-purchase">
+								<p>
+									{ translate( 'This site is managed through {{a}}Automattic for Agencies{{/a}}.', {
+										components: {
+											a: isA4A ? (
+												<a
+													href={ `https://agencies.automattic.com/sites/overview/${ site?.slug }` }
+												></a>
+											) : (
+												<strong></strong>
+											),
+										},
+									} ) }
+								</p>
+							</div>
+						) }
+						{ ! isAgencyPurchase && <PricingSection /> }
 						<PlanStorage
 							className="hosting-overview__plan-storage"
 							hideWhenNoStorage
 							siteId={ site?.ID }
 							StorageBarComponent={ PlanStorageBar }
 						>
-							{ storageAddons.length > 0 && (
+							{ storageAddons.length > 0 && ! isAgencyPurchase && (
```

Comment:
Let's check if we're loading before rendering this. It's now rendering then disappearing.